src/components/auth/PrivateRoute.js

```diff
@@ -12,9 +12,9 @@ export default function PrivateRoute({ exact, path, component }) {
     }
   }, [history, user]);
 
-  return (
+  return !!user ? (
```

Comment:
this check will make this route not show anything in the brief time you try to accees a private route when not logged in. Purely UI enhancement